dallinger/bots.py

```diff
@@ -8,6 +8,8 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
+from dallinger.config import get_config
+config = get_config()
 
```

Comment:
I don't think so, the entry points should take care of it. The tests are failing because they're missing the required config, which I'm pushing a change for now.